src/services/workflow.service.js

```diff
@@ -67,7 +67,6 @@ class WorkflowService {
     this.subscriptions = {}
 
     // mutations defaults
-    this.associations = null
```

Comment:
(This was never used since the commit that introduced it - https://github.com/cylc/cylc-ui/commit/e7a3fbed0aea527c472342c786ef64f64a2c5247#diff-5bf28b4f8de7d141f3dbfe760f5d05effbfc83e4debf8859ebee820fc81bfa7bR209)